.eslintrc

```diff
@@ -3,14 +3,15 @@
         "airbnb-base"
     ],
     "rules": {
-        "indent": ["error", 4],
         "no-console": "off",
-        "no-underscore-dangle": "off"
+        "no-underscore-dangle": "off",
```

Comment:
Why indent config is removed?

Reply:
Marking resolved as this question has been answered elsewhere.